package.json

```diff
@@ -2699,30 +2699,27 @@
     },
     "scripts": {
         "watch": "tsc -watch -p ./",
-        "build": "gulp build",
-        "ci-build": "gulp ci-build",
-        "package": "gulp package",
-        "ci-package": "gulp ci-package",
-        "lint": "gulp lint",
-        "test": "gulp test",
-        "test-only": "gulp test-only",
-        "unit-test": "gulp unit-test",
-        "webpack-dev": "gulp webpack-dev",
-        "webpack-prod": "gulp webpack-prod"
+        "build": "tsc -p ./",
```

Comment:
These scripts are sorta messy but this isn't even their final form. However, in order to avoid a PR that was super unwieldy, I made them compatible with our current build scripts. I intend to make these neater when we make the switch to `webpack --watch`.